subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -823,11 +823,16 @@ static void response_cb(struct http_response *rsp,
 		}
 
 		if (body_data != NULL) {
+			ret = mbedtls_md_update(&hb_context.dl.hash_ctx, body_data,
+					  body_len);
+			if (ret != 0) {
```

Comment:
Are you able to reproduce such error that needs mbedtls check? Is so, can you check whether `body_len > 0` before calling `mbedtls_md_update`? Perhaps the error is related to it?

Reply:
> Are you able to reproduce such error that needs mbedtls check?

I actually have never encounter such error before.

> Perhaps the error is related to it?

It is just that the coverity check is complaining that the return value of `mbedtls_md_update` isn't checked. Maybe casting the function to `void` can fix the complaint as well, but I think printing the error code isn't a bad idea either, in case somebody does indeed encounter some error.